### PR TITLE
for for practice we have created

### PR DESCRIPTION
**Pull request by miftahkk**
only for checking purpose

### CODE REVIEW

**Comment by minhaja3** [comment]
agin add two lines

File: newtest2 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+hi 
+hello 
+high 
+low
+very high
+now less
+line 112
```
Comment (minhaja3) [inline newtest2:7]: again add wo lines over in this file

Comment (miftahkk) [reply newtest2:7]: sure we will do this

File: newtest2 [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+ we are going to add multiple lines
+ this is line1
+ this is line2
```
Comment (minhaja3) [inline newtest2:10]: this is good

**Pull request merged**